### PR TITLE
allow run shows to work with an actual org id.

### PR DESCRIPTION
**Pull request by lyondhill**

### CODE REVIEW

File: task.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ type TaskService interface {
 	FindRuns(ctx context.Context, filter RunFilter) ([]*Run, int, error)
 
 	// Returns a single run
-	FindRunByID(ctx context.Context, id ID) (*Run, error)
+	FindRunByID(ctx context.Context, orgID, id ID) (*Run, error)
```
Comment (mark-rushakoff) [inline task.go]: Rename `id` to `runID` for clarity please.

File: task/backend/inmem_logreaderwriter.go [hunk 2]
```diff
@@ -154,7 +154,7 @@ func (r *runReaderWriter) ListLogs(ctx context.Context, logFilter platform.LogFi
 	defer r.mu.RUnlock()
 
 	if logFilter.Task == nil && logFilter.Run == nil {
-		return nil, errors.New("task or run is required")
+		return nil, errors.New("task and run is required")
```
Comment (mark-rushakoff) [inline task/backend/inmem_logreaderwriter.go]: `are` required.

Then it should be `||` on the line above, shouldn't it?

File: task/backend/store.go [hunk 3]
```diff
@@ -180,7 +180,7 @@ type LogReader interface {
 	ListRuns(ctx context.Context, runFilter platform.RunFilter) ([]*platform.Run, error)
 
 	// FindRunByID finds a run given a taskID and runID.
```
Comment (mark-rushakoff) [inline task/backend/store.go]: Update comment.

File: task/backend/storetest/logstoretest.go [hunk 4]
```diff
@@ -15,6 +15,7 @@ import (
 type CreateRunStoreFunc func(*testing.T) (backend.LogWriter, backend.LogReader)
 type DestroyRunStoreFunc func(*testing.T, backend.LogWriter, backend.LogReader)
 
+// NewRunStoreTest creates a testing swith for anything that implements the LogWriter and LogReader
```
Comment (mark-rushakoff) [inline task/backend/storetest/logstoretest.go]: A testing swith?

File: task/platform_adapter.go [hunk 5]
```diff
@@ -152,10 +152,10 @@ func (p pAdapter) FindRuns(ctx context.Context, filter platform.RunFilter) ([]*p
 	return runs, len(runs), err
 }
 
-func (p pAdapter) FindRunByID(ctx context.Context, id platform.ID) (*platform.Run, error) {
+func (p pAdapter) FindRunByID(ctx context.Context, orgID, id platform.ID) (*platform.Run, error) {
 	// TODO(lh): the inmem FindRunByID method doesnt need the taskId or orgId but we will need it PlatformAdapter
 	// this call to the store is a filler until platform.TaskService gets the update to add the IDs
```
Comment (mark-rushakoff) [inline task/platform_adapter.go]: Is this TODO removable now?

File: go.sum [hunk 6]
```diff
@@ -283,6 +284,8 @@ golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
 golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 h1:+DCIGbF/swA92ohVg0//6X2IVY3KZs6p9mix0ziNYJM=
 golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
 golang.org/x/tools v0.0.0-20180904205237-0aa4b8830f48/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
+golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e h1:FDhOuMEY4JVRztM/gsbk+IKUQ8kj74bxZrgw87eMMVc=
+golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
```
Comment (mark-rushakoff) [inline go.sum]: Not sure why we now have three versions of x/tools. Does this go down to one copy if you `go mod tidy`?

File: task/platform_adapter.go (hunk 5 above)
Comment (lyondhill) [reply task/platform_adapter.go]: Yes!

File: task/backend/storetest/logstoretest.go (hunk 4 above)
Comment (lyondhill) [reply task/backend/storetest/logstoretest.go]: oo Yah that doesn't make sense.

**Pull request merged**